storage/innobase/log/log0crypt.cc

```diff
@@ -221,8 +221,8 @@ ATTRIBUTE_COLD bool log_decrypt(byte* buf, lsn_t lsn, ulint size)
 		ut_ad(LOG_CRYPT_HDR_SIZE + dst_size
 		      == 512 - LOG_BLOCK_CHECKSUM - LOG_BLOCK_KEY);
 
-		uint dst_len;
-		int rc = encryption_crypt(
+                uint dst_len= (uint) dst_size;
+                int rc = encryption_crypt(
 			buf + LOG_CRYPT_HDR_SIZE, static_cast<uint>(dst_size),
```

Comment:
Please do not add C-style casts and do not replace TABs with spaces.

Reply:
Thanks, not sure why clang-tidy did that formatting

Reply:
I think this should be all set